ui/src/components/device/DeviceAdd.vue

```diff
@@ -55,13 +55,6 @@
           Close
         </v-btn>
       </v-card-actions>
-
-      <v-snackbar
-        v-model="copySnack"
-        :timeout="3000"
-      >
```

Comment:
When I copy the add device command, a notification of the "tenantID" appears
in it's  place.